CompanyJobTitles.md

```diff
@@ -0,0 +1,64 @@
+Company Job Titles
+==================
+
+#### Table of Contents
+_______
+
+- [Summary](#summary)
+- [Request structure](#request-structure)
+- [Response structure](#response-structure)
+- [Versioning](#versioning)
+
+## Summary
+
+The company Job Titles service provides a list of job titles (along with their ONet and Carotene 
+classifications) that are personalized for a given company. Company Job Titles is available at 
+`/core/company/jobtitles`.
+
+
+## Request Structure
+Requests consist of a `company_id` string:
+
+```json
+{
+	"company_id": "NC6623058c-2e0b-4998-ac4a-72e478e91337",
+	"carotene_version": "carotenev3",
+	"onet_version": "onet17"
+}
+```
+
+Available taxonomies are `onet17` for `onet_version` and `carotenev3` for `carotene_version`.
```

Comment:
this info is duplicated here - perhaps replace with something like "The ONet and Carotene versions currently supported are listed in the Versioning[#Versioning] section of this document and are subject to change over time. Any removal of support for a taxonomy version will be implemented as a versioned change to the API itself."

[side note: should we do that second thing? I feel like probably?]

Reply:
Changed